src/main/java/com/PhamKornbluhGroup/runner.java

```diff
@@ -0,0 +1,7 @@
+package com.PhamKornbluhGroup;
+
+public class runner {
+    public static void main(String[] args) {
+        new PublicStashTabBulkAPICaller().callAndPrintResults();
```

Comment:
Consider initializing the object with a variable name and then invoke callAndPrintResults on the next line. This is for debugging purposes.

Reply:
Done. updated to 2 lines
```
GGGAPIHandler handler = new GGGAPIHandler();
handler.callAndPrintResults();
```